src/app/contact-us/contact-us.component.html

```diff
@@ -1,81 +1,60 @@
 <div id="ContactUsPage">
```

Comment:
Can you add `class="body"` to this `div` that way when the font is merged the text would be Actor.